src/action/main.ts

```diff
@@ -76,14 +76,14 @@ async function run(): Promise<void> {
         '--disable-sandbox',
         inputs.steward.extraArgs?.value.split(' ') ?? [],
       ], inputs.steward.extraJars)
-
+    } finally {
       if (files.existsSync(workspace.runSummary_md)) {
         logger.info(`✓ Run Summary file: ${workspace.runSummary_md}`)
 
         const summaryMarkdown = files.readFileSync(workspace.runSummary_md, 'utf8')
         await core.summary.addRaw(summaryMarkdown).write()
```

Comment:
A failure in these 2 lines would not not get cleaned up. IS this a problem ? (I know too little about this)